api/spec/requests/users/sessions_controller_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 RSpec.describe Users::SessionsController, type: :request do
   describe 'POST /users/login' do
-    let!(:user) { create :user, password: 'CorrectPassword#123' }
+    let!(:user) { create :user, password: 'CorrectPassword#123', confirmed_at: Time.zone.now }
+    let!(:unconfirmed_user) { create :user, password: 'CorrectPassword#123' }
```

Comment:
Dado que solo usamos este `unconfirmed_user` en uno de los contexts (`when user is not confirmed`), no conviene moverlo dentro de ese bloque?

```ruby
context 'when user is not confirmed' do
      let!(:unconfirmed_user) { create :user, password: 'CorrectPassword#123', confirmed_at: nil }
      let(:password) { 'CorrectPassword#123' }

      ....
end
```